docs/Feature-Broadcasting.md

```diff
@@ -1,12 +1,19 @@
 # Using the Broadcast Feature
-The Player, Heuristic and Internal brains have been updated to support broadcast. The broadcast feature allows you to collect data from your agents in python without controling them.  
+
+The Player, Heuristic and Internal brains have been updated to support broadcast. The broadcast feature allows you to collect data from your agents in using a Python program without controlling them.  
+
 ## How to use : Unity
+
 To turn it on in Unity, simply check the `Broadcast` box as shown bellow:
 
 ![Broadcast](images/broadcast.png)
 
 ## How to use : Python 
-When you launch your Unity Environment from python, you can see what the agents connected to non-external brains are doing. When calling `step` or `reset` on your environment, you retrieve a dictionary from brain names to `BrainInfo` objects. Each `BrainInfo` the non-external brains set to broadcast.  
+
+When you launch your Unity Environment from a Python program, you can see what the agents connected to non-external brains are doing. When calling `step` or `reset` on your environment, you retrieve a dictionary from brain names to `BrainInfo` objects. Each `BrainInfo` the non-external brains set to broadcast.  
```

Comment:
"Each `BrainInfo` the non-external brains set to broadcast."  ... not sure what that means

Reply:
fixed.